backends-clickhouse/src/test/scala/io/glutenproject/execution/GlutenClickHouseTPCDSParquetColumnarShuffleAQESuite.scala

```diff
@@ -43,6 +45,10 @@ class GlutenClickHouseTPCDSParquetColumnarShuffleAQESuite
       //      .set("spark.sql.files.openCostInBytes", "134217728")
       .set("spark.sql.adaptive.enabled", "true")
       .set("spark.memory.offHeap.size", "4g")
+      .set(
+        GlutenConfig.GLUTEN_LIB_PATH,
+        "/home/liangjiabiao/workspace/github/gluten/cpp-ch/"
```

Comment:
why needs add this config? can use mvn options to set this config.

Reply:
Ignore this. There is a bug for configuring this on local, but no time to debug it.